events: fix slow client connection to empty event stream (#10637)
hashicorp/nomad: CHANGELOG.md

BUG FIXES:
* cli: Fixed a bug where `quota status` and `namespace status` commands may panic if the CLI targets a pre-1.1.0 cluster

> * Event stream: Fixed slow event stream connection initialisation not having any events yet [[GH-10636](https://github.com/hashicorp/nomad/issues/10636)]

Reviewer: Small nit-pick, we use the `api:` prefix for api endpoints
```suggestion
* api: Fixed event stream connection initialization when there are no events to send [[GH-10636](https://github.com/hashicorp/nomad/issues/10636)]
```

Author: Thank you very much for the super quick review! :)
